src/handbook/product/strategy.md

```diff
@@ -4,59 +4,44 @@ navTitle: Strategy
 
 # Product Strategy
 
-Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowForge’s vision. It’s a draft, as everything in the handbook is a draft, so we will revisit regularly and iterate when there is a reason to do so. FlowForge is a 100 percent agile company, so our product is; timing is everything. Have a look at our roadmap for what we're pursuing in the upcoming releases. 
+Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowForge’s vision, to become the leading "Low-Code IIoT Application Platform".
+It’s a draft, as everything in the handbook is a draft, so we will revisit regularly and iterate when there is a reason to do so. FlowForge is a 100 percent agile company, so our product is; timing is everything.
+Have a look at our [roadmap](https://flowforge.com/product/roadmap/) for what we're pursuing in the upcoming releases.
 
-### FlowForge for enterprise
+### Advancing Enterprise-Readiness
 
-The cornerstone of FlowForge is to make Node-RED more accessible to enterprises. Therefore, we are constantly implementing requirements that support the easy use of Node-RED in production environments. We will focus on:
+FlowForge is committed to augmenting the enterprise-readiness of Node-RED. Our strategy emphasizes enhancing audit and compliance capabilities, instituting stringent preventive and corrective controls, and bolstering security. By integrating these vital aspects, we aim to create a reliable, secure, and scalable environment. This initiative underlines our dedication to surpassing enterprise expectations and strengthening Node-RED's position in the industry.
```

Comment:
```suggestion
FlowForge is committed to augmenting the enterprise-readiness of Node-RED. Our strategy emphasises enhancing audit and compliance capabilities, instituting stringent preventive and corrective controls, and bolstering security. By integrating these vital aspects, we aim to create a reliable, secure, and scalable environment. This initiative underlines our dedication to surpassing enterprise expectations and strengthening Node-RED's position in the industry.
```

Reply:
According to https://flowforge.com/handbook/marketing/website/#website
"All written content should be in UK English." For me, both options are fine, I just need to know.